_build_cfg.yml

```diff
@@ -32,6 +32,18 @@ Topics:
     File: ipa_2fa
 
 ---
+Name: Installation
+Dir: install
+Topics: 
+  - Name: Installing on RHEV
+    Dir: installing_on_rhev
+    Topics: 
+      - Name: Installing the Appliance
+      - File: Installing_Appliance
```

Comment:
Hi Deon, the 2 File entries here shouldn't have the - prefix.  asciibinder fails to build with those.